doc/guide/installation.rst

```diff
@@ -38,6 +44,11 @@ The easiest way to keep your suite updated is to use
           # ...
           extensions:
             SensioLabs\Behat\PageObjectExtension: ~
+            Behat\MinkExtension:
```

Comment:
usually symfony yamls leave 4 spaces as tab. Do we have a standard for this behat.yml?